src/Frontend/Page/Overview.hs

```diff
@@ -174,26 +174,24 @@ instance ToHtml Tabs where
 
 -- * idea lists
 
-data ListItemIdeaContext
+data ListItemIdeaContext  -- TODO: rename to 'ListLocation'?  'Context' is used for too many things.
     = IdeaInIdeasOverview
     | IdeaInViewTopic
     | IdeaInUserProfile
   deriving (Eq, Show, Read, Generic)
 
 data ListItemIdea = ListItemIdea
-      { _listItemIdeaContext    :: ListItemIdeaContext
-      , _listItemIdeaPhase      :: Maybe Phase
-      , _listItemIdeaNumVoters  :: Int
-      , _listItemIdea           :: Idea
-      , _listItemRenderContext  :: RenderContext
+      { _listItemRenderContext  :: RenderContext
+      , _listItemIdeaContext    :: ListItemIdeaContext
```

Comment:
Perhaps Position? There is already ` data IdeaLocation`. Also, we don't want to mix it up with Space.